src/pages/home/report/ReportActionCompose.js

```diff
@@ -578,6 +581,10 @@ class ReportActionCompose extends React.Component {
                                 ]}
                                 onPress={this.submitForm}
                                 underlayColor={themeColors.componentBG}
+
+                                // Keep focus on the composer when Send message is clicked.
+                                // eslint-disable-next-line react/jsx-props-no-multi-spaces
```

Comment:
We should disable this eslint rule

Reply:
Yeah, I guess so. It is used in many places